### PR TITLE
Rough draft

### PR DESCRIPTION
**Pull request by dicgf8**

### CODE REVIEW

File: src/components/tstFormFields/demoBasicInputUsage/script.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+angular.module('tstInputBasicDemo', ['ngMaterial', 'ngMessages'])
+	.controller('DemoCtrl', ['$scope', function($scope){
+		
+	}])
+	.config(function($mdThemingProvider){
+		console.log($mdThemingProvider);
+
+		$mdThemingProvider.definePalette('standardPalette', {
```
Comment (qrpike) [inline src/components/tstFormFields/demoBasicInputUsage/script.js]: remove?

File: src/components/tstFormFields/formFields.scss [hunk 2]
```diff
@@ -0,0 +1,240 @@
+$PRIMARY: #4d91db;
```
Comment (qrpike) [inline src/components/tstFormFields/formFields.scss]: Remove Variables

File: src/core/core.js [hunk 3]
```diff
@@ -36,10 +37,94 @@ function MdCoreConfigure($provide, $mdThemingProvider) {
...
+  });
+
   $mdThemingProvider.theme('default')
-    .primaryPalette('indigo')
-    .accentPalette('pink')
-    .warnPalette('deep-orange')
+    .primaryPalette('customPrimary')
+    .accentPalette('customAccent')
+    .warnPalette('customWarn')
     .backgroundPalette('grey');
```
Comment (qrpike) [inline src/core/core.js]: grey > customBackground

File: src/core/style/themes.scss [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 html, body {
   &.md-THEME_NAME-theme {
-    color: '{{foreground-1}}';
+    color: '{{background-900}}';
```
Comment (qrpike) [inline src/core/style/themes.scss]: Keep the foreground, and edit the theme to contrastDark/Light colors

File: src/core/style/tst-theme/variables.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
+/* Overides for AM variables */
+
+$font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
+$font-size:   14px;
+
+/* TST variables */
+
+$PRIMARY: #4d91db;
```
Comment (qrpike) [inline src/core/style/tst-theme/variables.scss]: Remove un-needed vars

File: src/formFields/formFields.scss [hunk 6]
```diff
@@ -0,0 +1,239 @@
+$PRIMARY: #4d91db;
```
Comment (qrpike) [inline src/formFields/formFields.scss]: Not in the components folder?

File: src/formFields/js/inputDirective.js [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+	.directive('tsInput', TsInput);
+
+
+/**
+ * @ngdoc directive
+ * @name tsInput
+ * @module tsstrap.formFields
+ *
+ * @description
+ * `<ts-input>` is a common component to match the required stules
```
Comment (DevVersion) [inline src/formFields/js/inputDirective.js]: `styles`

File: src/components/tstFormFields/demoBasicInputUsage/script.js (hunk 1 above)
Comment (dicgf8) [inline src/components/tstFormFields/demoBasicInputUsage/script.js]: Removed

File: src/components/tstFormFields/formFields.scss (hunk 2 above)
Comment (dicgf8) [inline src/components/tstFormFields/formFields.scss]: Removed

File: src/core/core.js (hunk 3 above)
Comment (dicgf8) [inline src/core/core.js]: changed

File: src/core/style/themes.scss (hunk 4 above)
Comment (dicgf8) [inline src/core/style/themes.scss]: Reverted

File: src/core/style/tst-theme/variables.scss (hunk 5 above)
Comment (dicgf8) [inline src/core/style/tst-theme/variables.scss]: Removed

File: src/formFields/formFields.scss (hunk 6 above)
Comment (dicgf8) [inline src/formFields/formFields.scss]: Removed

File: src/formFields/js/inputDirective.js (hunk 7 above)
Comment (dicgf8) [inline src/formFields/js/inputDirective.js]: Completed

**Comment by qrpike** [comment]
LGTM

**Pull request merged**